mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,9 +3,17 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
-		var TPL_TASK = '<li>' +
-				'{task}' +
-				'<i class="icon-remove"></i>' +
+		var TPL_FINISHED_TASK = '<li> ' +
+			'{taskFinished}' +
+				'<button>' +
+					'<i class="icon-trash"></i>' +
+				'</button>' +
+			'</li>';
+
+		var TPL_TASK = '<li>{task} ' +
```

Comment:
```{task}``` is a child text node, so it should be in it's own row, just like ```{taskFinished}``` above. Both should be indented the same as sibling nodes. 

What does this lebel represent? The word ```task``` is not clear. Task name? Title? Content?